### ISSUE TITLE
Hatch/Lid Improvements

### ISSUE THREAD

**Issue by StephenCarlson** [StephenCarlson/MiniHawk-VTOL#15]
The Hatch parting-plane is arbitrarily defined, and should be explicitly defined (The slice is at some transcendental angle, and should be explicitly defined to a nice round integer degree). A collateral wish-list item to this is to explicitly define the lip and hole geometry, much like how kayak spray-skirt cockpits/keyholes are standardized. 

The current lip seal isn't very good, and the latch/catch geometry can be improved.

OML isn't changed with this issue, but the compatibility from v1.0.x forward is broken.

**Comment by StephenCarlson** [comment]
Much better defined in ede11e9. The parting plane is parallel to the aircraft XY coordinate plane, and is 22.0mm offset.

Also, the Hatch definition is released as a STP/STEP file for modification in other CAD systems. This should allow the community to create variants for FPV cameras and other accessories. This and other released CAD definitions have the file name ending as _RELEASED.stp .

**Comment by StephenCarlson** [comment]
I think I've reached perfection in the hatch retention method. Over the previous weekend, "for fun", I designed, prototyped, printed and perfected a latch-lock lever/hook mechanism. It uses the old 1.8x24mm nacelle pins for the hinge, so no need to cut new metal pins. Detents hold the lever in the secured position, but I suppose magnets could also be used in a future design, but I'm adverse to this as it would put strong uncontained magnetic fields in close proximity to the flight controller magnetometer. Pushed at 356c2511adc2a96dd797cd49411159d223af6fd2 . I think it is safe to close this issue and assign to Version 2.1.

![image](https://user-images.githubusercontent.com/6984010/143158582-56cab7be-f9dd-4616-a0e7-a50048bce03b.png)
